authors/apps/authentication/tests/test_authentication.py

```diff
@@ -23,6 +24,7 @@ def test_login_for_signed_up_user(self):
         
         self.assertEquals("test@testuser.com", self.logged_in_user.data["email"])
```

Comment:
@Emmanuelaaron instead of typing in `test@testuser.com` manually, I am suggesting you just reference it since it exists in the test_base file for better maintainability.